### PR TITLE
 chore: `just sign-release v0.3.0-rc.0` by dpc 

### PR DESCRIPTION
**Pull request by dpc**

### CODE REVIEW

File: releases/v0.3.0-rc.0-x86_64-linux.SHA256SUMS [hunk 1]
```diff
@@ -0,0 +1,10 @@
+9733ff0737b52b7042307786d9a706300e46d7f48db8acfe1b9beeaa1ca8b59e  v0.3.0-rc.0-x86_64-linux/fedimint-cli
```
Comment (elsirion) [inline releases/v0.3.0-rc.0-x86_64-linux.SHA256SUMS:1]: I'm getting the same hashes

**Review by elsirion — DISMISSED** [review]

**Comment by justinmoon** [comment]
Running from `0b8cdd855cc46e58a74fa04678849bddd8da3832` 

```
$ just sign-release v0.3.0-rc.0
./scripts/release/sign.sh v0.3.0-rc.0
Building...
warning: not writing modified lock file of flake 'github:dpc/bundlers/e8aafe89a11ae0a5f3ce97d1d7d0fcfb354c79eb':
• Updated input 'nix-bundle':
    'github:dpc/nix-bundle/ec52fce86513d73e989c6cff0367956c6b07a1ba' (2024-02-21)
  → 'github:dpc/nix-bundle/8ab9acfe0a31805de7899eddb8f3312d4e34a13d' (2024-02-21)
error:
       … in the left operand of the update (//) operator

         at /nix/store/f5ykcnsjzkvri0nmcj64d221lkxhrzrf-source/flake.nix:52:19:

           51|                 inherit system;
           52|               })) // (if drv?program then { } else {
             |                   ^
           53|                 name =

       … <borked>

         at «none»:0: (source not available)

       (stack trace truncated; use '--show-trace' to show the full trace)

       error: attribute 'aarch64-darwin' missing

       at /nix/store/18amc9dx7hgg3pqd70mzk9azaxap748i-source/flake.nix:12:20:

           11|       nix-bundle = { program, system }: let
           12|         nixpkgs' = nixpkgs.legacyPackages.${system};
             |                    ^
           13|         nix-bundle = import self { nixpkgs = nixpkgs'; };
error: Recipe `sign-release` failed on line 128 with exit code 1
```

**Comment by dpc** [comment]
@justinmoon Bundling of debs and rpms not supported on Darwin. I guess we'll have to skip it so at least raw binaries can be signed over.

**Comment by dpc** [comment]
@justinmoon fixed

**Review by elsirion — APPROVED** [review]

**Comment by elsirion** [comment]
Do we expect this to output anything useful in the first place? We do not publish mac binaries right now, so also don't really need signatures for them. Do we expect cross-compiled binaries to match and do we even support mac->linux cross compilation?

**Comment by justinmoon** [comment]
@elsirion I'm not sure. I just try to run these PRs as I'm able and report what happens on Mac.

**Comment by dpc** [comment]
@justinmoon Can you

```
bash -x ./scripts/release/sign.sh v0.3.0-rc.0
```

**Comment by dpc** [comment]
Anyway, let's land, macos can wait.

**Review by justinmoon — APPROVED** [review]

**Comment by fedimint-backports** [comment]
Backport failed for `releases/v0.3`, because it was unable to cherry-pick the commit(s).

Please cherry-pick the changes locally and resolve any conflicts.
```bash
git fetch origin releases/v0.3
git worktree add -d .worktree/backport-4627-to-releases/v0.3 origin/releases/v0.3
cd .worktree/backport-4627-to-releases/v0.3
git switch --create backport-4627-to-releases/v0.3
git cherry-pick -x 49be761d2ec089852b3f0e86fb0a4a39883ec5c5 eae958bdbaac10b016859ea81f398af7ac9851de 600c1fb5cf21ac4014e9e6c1732c6e3bd4a28008 4d5c7f20b4018f621801535b5b23aa9234ed6c80 e324f0f9398b6071495b64b2315bc2c735b719b8
```

**Pull request merged**